modules/ROOT/pages/create-or-modify-a-page.adoc

```diff
@@ -1,65 +1,178 @@
+
+. From these tabs, click on the *_Edit_* icon at the right of the _Target form_ field. This opens the UI Designer on a new form, with some widgets, containers, and variables already created to speed up the design process, using the contract inputs.
+. Rename it
+. Customize what needs to be displayed as well as the _Target URL on success_ of the *_Submit_* button.
+
+For more technical information about the forms, visit xref:forms-development.adoc[this dedicated documentation page].
+
+== Create a page
+
+=== From scratch
+. From Bonita Studio coolbar, click on the *_UI Designer_* icon to open the UI Designer. +
+. Click on the *_Create_* button
+. In the pop up, choose the element to create
+. Give it a name
+. Click on the *_Create_* button
+. In the page editor, drag and drop the widgets, fragments, or BDM objects to create the right user interface
+
+=== From a Bonita provided page
+
+Only Bonita pages created with the UI Designer can be customized in the UI Designer. Indeed, as Bonita applications have evolved in several phases, so did the frameworks in which their pages have been made.
+
+==== List of Bonita Applications customizable pages
+
+===== Bonita User Application
+
+In this application, you can customize:
+
+* Bonita User Case List `UserCaseListBonitaV_`
+* Bonita User Case Details `UserCaseDetailsBonitaV_`
+
+==== Bonita Super Administrator and Administrator Applications
+
+In thess applications, when relevant, you can customize:
+
+* Bonita Admin Process List `adminProcessListBonitaV_`
+* Bonita Admin Case Details `adminCaseDetailsBonitaV_`
+* Bonita Admin Task List `adminTaskListBonitaV_`
+* Bonita Admin Task Details `adminTaskDetailsBonitaV_`
+* Bonita Admin User List `adminUserListBonitaV_`
+* Bonita Admin Group List `adminGroupListBonitaV_`
+* Bonita Admin Role List `adminRoleListBonitaV_`
+* Bonita Admin Install Export Organization `adminInstallExportOrganizationBonitaV_`
+* Bonita Admin Profile List `adminProfileListBonitaV_`
+* Bonita Admin Resources List `adminResourceListBonitaV_`
+
+The "_" in "V_" stands for the latest "version" of the resource.
 
-After a page, layout, form, fragment is created, you can update it in the *Page editor*. Update your page, form or fragment by dragging and dropping widgets from the palette to the whiteboard. Edit widget properties in the widget properties panel, create data, and bind it to the widgets.
+[NOTE]
+====
+
+All the "Details" pages must be used as "orphan" pages in the application descriptor, since they need an id to be passed in order to display the details of the right item. They cannot be used at top-level menu.
+====
+
+For more technical information about the pages, visit xref:pages-development.adoc[the dedicated documentation page]. 
+
+== Create a layout
+
+=== From the provided Bonita layout
+==== If you have not installed any Bonita application in your project in Bonita Studio 
```

Comment:
I would have put the 'positive statement' first (_if you have installed a Bonita application_).

Reply:
ok